src/WPF/WPF.Viewer/Samples/Map/CreateDynamicBasemapGallery/CreateDynamicBasemapGallery.xaml

```diff
@@ -0,0 +1,44 @@
+﻿<UserControl x:Class="ArcGIS.WPF.Samples.CreateDynamicBasemapGallery.CreateDynamicBasemapGallery"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013">
+    <Grid>
+        <esri:MapView x:Name="MyMapView" />
+        <Border Style="{StaticResource BorderStyle}">
+            <StackPanel>
+                <ProgressBar x:Name="ProgressBar"
+                             Height="25"
+                             IsIndeterminate="True" />
+                <StackPanel x:Name="DynamicBasemapStyleGallery"
+                            Margin="5"
+                            Visibility="Collapsed">
+                    <ListView x:Name="BasemapStyleGallery" MaxHeight="500">
```

Comment:
For this and other list views, you should probably set SelectionMode to "Single" -- it doesn't make sense to let users select multiple basemaps.